Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -402,6 +403,21 @@ WarpXOpenPMDPlot::~WarpXOpenPMDPlot ()
   }
 }
 
+
+void WarpXOpenPMDPlot::flushCurrent(bool isBTD) const
+{
+     WARPX_PROFILE("WarpXOpenPMDPlot::flushCurrent");
+     // open files from all processors, in case some will not contribute below
+     // m_Series->flush(); ## NOTE flush only flushes if data is dirty. When the underlying function is collective
+     //                    ## like PDW, there will be traouble.
+     // the change will be use PDW when not BTD, and Put if BTD to avoid slowing down due to  multiple writes of small data
+     openPMD::Iteration currIteration = GetIteration(m_CurrentStep, isBTD);
+     if (isBTD)
+         currIteration.seriesFlush(  "adios2.engine.preferred_flush_target = \"buffer\"" );
+     else
+         currIteration.seriesFlush();
```

Comment:
```suggestion
     if (isBTD) {
         currIteration.seriesFlush(  "adios2.engine.preferred_flush_target = \"buffer\"" );
     }
     else {
         currIteration.seriesFlush();
     }
```
Could you please add braces here?